src/ws/cockpitauth.c

```diff
@@ -1135,74 +893,38 @@ cockpit_auth_remote_login_async (CockpitAuth *self,
                                  gpointer user_data)
 {
   GSimpleAsyncResult *task;
-  CockpitCreds *creds = NULL;
   AuthData *ad = NULL;
-  RemoteLoginData *rl;
-  const gchar *password;
-  gchar *csrf_token;
   GBytes *input;
+  GBytes *pw_bytes;
   gchar *user = NULL;
+  const gchar *password;
+  const gchar *command;
+
+  const gchar *argv[] = {
+      cockpit_ws_ssh_session_program,
+      "--user",
+      "user",
+      "--ignore-hostkey",
+      "--expect-auth-data",
```

Comment:
--ignore-hostkey is only always used here. When used with sshtransport that is usually going to be off.
I'll switch --expect-auth-data to --no-auth-data.